components/input/NumericInput.js

```diff
@@ -0,0 +1,132 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { IconChevronUpSmallOutline, IconChevronDownSmallOutline } from '@teamleader/ui-icons';
+import Icon from '../icon';
+import InputBase from './InputBase';
+import theme from './theme.css';
+
+class SpinnerControls extends PureComponent {
+  render() {
+    const { inverse, handleOnSpinnerUpClick, handleOnSpinnerdownClick } = this.props;
+    const iconProps = {
+      color: inverse ? 'teal' : 'neutral',
+      element: 'button',
+      tint: inverse ? 'lightest' : 'darkest',
+    };
+
+    return (
+      <div className={theme['spinner']}>
+        <Icon className={theme['spinner-up']} onClick={() => handleOnSpinnerUpClick()} {...iconProps}>
+          <IconChevronUpSmallOutline />
+        </Icon>
+        <Icon className={theme['spinner-down']} onClick={() => handleOnSpinnerdownClick()} {...iconProps}>
```

Comment:
PP
```suggestion
        <Icon className={theme['spinner-down']} onClick={handleOnSpinnerdownClick} {...iconProps}>
```

Reply:
Also PP that the function name here becomes `handleOnSpinnerDownClick` because `Up` is also capitalized for the other function.